README.windows.rst

```diff
@@ -41,9 +41,262 @@ Current status
 The rados and rbd binaries and libs compile successfully and can be used on
 Windows, successfully connecting to the cluster and consuming pools.
 
+Ceph filesystems can be mounted using the ``ceph-dokan`` command, which
+requires the Dokany package to be installed. Note that dokany is a well
+maintained fork of the Dokan project, allowing filesystems to be implemented
+in userspace, pretty much like Fuse.
+
+RBD images can be mounted using the ``rbd`` or ``rbd-wnbd`` commands. The
+``WNBD`` driver is required for mapping RBD images on Windows.
+
 The libraries have to be built statically at the moment. The reason is that
 there are a few circular library dependencies or unspecified dependencies,
 which isn't supported when building DLLs. This mostly affects ``cls`` libraries.
 
-A significant number of tests from the ``tests`` directory have been port,
+A significant number of tests from the ``tests`` directory have been ported,
 providing adequate coverage.
+
+Supported platforms
+===================
+
+Windows Server 2019 and Windows Server 2016 are supported. Previous Windows
+Server versions, including Windows client versions such as Windows 10, might
+work but haven't been tested.
+
+Windows Server 2016 does not provide unix sockets, in which case the Ceph admin
+socket feature will be unavailable.
+
+Compatibility
+=============
+
+RBD images can be exposed to the OS and host Windows partitions or they can be
+attached to Hyper-V VMs in the same way as iSCSI disks.
+
+At the moment, the Microsoft Failover Cluster refuses to use WNBD disks as
```

Comment:
Just for my own curiosity, any more background on this limitation?

Reply:
We don't support SCSI Persistent Reservations yet. I think we can implement it using ceph locks. Most probably we'll take care of that in the upcoming months since there's significant demand for this feature.

I'll update the readme, adding some more details on this.